apps/site/assets/webpack.config.export-headerfooter.js

```diff
@@ -103,9 +103,11 @@ module.exports = (env, argv) => {
       // copy images and fonts
       new CopyWebpackPlugin({ patterns: [
             { from: "static/fonts/*", to: "fonts/[name][ext]" },
-            { from: "static/favicon.ico", to: "favicon.ico" },
+            { from: "static/favicon.ico", to: "favicon.ico" },        
```

Comment:
nit-pick: This seems like accidental whitespace

Reply:
Fixed